Examples/Scripts/Python/full_chain_odd_LRT.py

```
@@ -15,11 +15,13 @@
     ParticleConfig,
     ParticleSelectorConfig,
     addPythia8,
+    ParticleSelectorConfig,
```

Ruff (0.8.2)

Redefinition of unused `ParticleSelectorConfig` from line 16

Remove definition: `ParticleSelectorConfig`

(F811)